docs/Biconomy AA Stack/Account/description.md

```diff
@@ -2,7 +2,10 @@
 sidebar_position: 1
 ---
 # Description
+Add & customise programmble modules to extand certain Smart Account functionality. Then build & send the relevant UserOps. 
 
 Building and sending UserOperations is a key offering of any toolkit designed for ERC4337. The Biconomy account package stands as an exemplary toolkit in this regard. Meticulously crafted with developers' needs in mind, this package seamlessly integrates the essential features associated with ERC-4337. It simplifies the process of creating and sending UserOperations, thus optimizing the development and management of decentralized applications (dApps).
 
-The Biconomy account package achieves this by providing a comprehensive set of methods that enable developers to effortlessly create UserOperations. Combined with the sophisticated backend infrastructure of the Biconomy platform, it ensures efficient and reliable transmission of these operations across EVM networks.
\ No newline at end of file
+The Biconomy account package achieves this by providing a comprehensive set of methods that enable developers to effortlessly create UserOperations. Combined with the sophisticated backend infrastructure of the Biconomy platform, it ensures efficient and reliable transmission of these operations across EVM networks.
+
+The Biconomy account package also supports the addition of modules. Each module unlocks new Smart Account funcationality such as passkeys, session keys, automation etc. 
```

Comment:
instead of automation, we should add other features like account recovery.... coz automation is not a module, its a use case that can be implemented using session key module for example.